src/scenario_perception_pipeline_benchmark_main.cpp

```diff
@@ -0,0 +1,11 @@
+#include "moveit_middleware_benchmark/scenarios/scenario_perception_pipeline.hpp"
```

Comment:
Add license

Reply:
Fixed in https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/5/commits/dafc402a5f4ed41de6159a366e71677f5feea573 commit